src/Plugins/ComplexCore/src/ComplexCore/Filters/ImportVolumeGraphicsFileFilter.cpp

```diff
@@ -0,0 +1,558 @@
+#include "ImportVolumeGraphicsFileFilter.hpp"
+
+#include <fstream>
+
+#include "Algorithms/ImportVolumeGraphicsFile.hpp"
```

Comment:
This should be the complete path to the Algorithms folder. so ComplexCore/Filters/Algorithms/.....